### PR TITLE
✅ updated test for outlook.

### PR DESCRIPTION
**Pull request by Diego-H-S**
## Summary
More test have been added to the file `test_outlook.py`

## Importance
low

## Checklist

This PR:

- [x] adds new tests (if appropriate)
- [ ] updates docstrings for any new functions or function arguments (if appropriate)
- [ ] updates `CHANGELOG.md` with a summary of the changes

### CODE REVIEW

File: tests/integration/test_outlook.py [hunk 1]
```diff
@@ -1,14 +1,62 @@
...
+
     outlook = Outlook(
         mailbox_name=outlook_env_vars["mail_example"],
+        credentials=outlook_env_vars,
         start_date="2022-04-28",
         end_date="2022-04-29",
     )
     df = outlook.to_df()
+
     assert isinstance(df, pd.DataFrame)
```
Comment (Rafalz13) [inline tests/integration/test_outlook.py:34]: You can also check what is inside or columns in DF

File: viadot/sources/outlook.py [hunk 2]
```diff
@@ -4,9 +4,9 @@
 import pandas as pd
 from O365 import Account
 
-from ..config import local_config
-from ..exceptions import CredentialError
-from .base import Source
+from viadot.config import local_config
+from viadot.exceptions import CredentialError
+from viadot.sources.base import Source
```
Comment (Rafalz13) [inline viadot/sources/outlook.py:9]: Were these changes necessary?

**Review by Rafalz13 — COMMENTED** [review]
Added few comments

File: viadot/sources/outlook.py (hunk 2 above)
Comment (Diego-H-S) [reply viadot/sources/outlook.py:9]: they are not, but I added them here to standardize the import way. Better absolute import instead of relative one. In any case, this doesn't affect to the code.

File: tests/integration/test_outlook.py [hunk 3]
```diff
@@ -1,14 +1,62 @@
...
+
+    assert isinstance(outlook.credentials, dict)
+
+
+@mock.patch("O365.Account", return_value=None)
+@mock.patch("O365.Account.authenticate", return_value="trial1")
+@mock.patch("O365.Account.mailbox", return_value="trial2")
+@mock.patch("viadot.sources.Outlook.to_df", return_calue=to_df)
+@pytest.mark.task
+def test_outlook_tas(mock_account, mock_auth, mock_mail, mock_to_df):
```
[the remark this answers is not in the document]
Comment (Diego-H-S) [reply tests/integration/test_outlook.py:53]: done!

**Review by Rafalz13 — APPROVED** [review]
